CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 - Optimize `TargetNode`'s set operations https://github.com/tuist/tuist/pull/1095 by @kwridan
 - Optimize `BuildPhaseGenerator`'s method of detecting assets and localized files https://github.com/tuist/tuist/pull/1094 by @kwridan
 - Concurrent project generation https://github.com/tuist/tuist/pull/1096 by @kwridan
+- Add possibility to skip generation of default schemes https://github.com/tuist/tuist/pull/1130 by @olejnjak
```

Comment:
Just a nitpick - I believe this belongs to a `## Next` section

Reply:
Good catch! Merging master obviously created this issue 👍